### PR TITLE
Fixed labels on charts, Enabled running multiple LF profiles before CBA, Cleanup Frontend Files

### PR DESCRIPTION
**Pull request by avpeery**
This PR completes a couple of remaining to-dos. It allows for running/saving multiple load forecast profiles before running the profile through the cost benefit analysis. It cleans up the frontend files (uniform naming, added error handling to load control and used reusable alert for error handling in load forecast). Implemented revoke celery task if max retries causes failure callback (without this, it continues running/attempting to run the celery task). This PR also fixes overlapping chart labels and removes inaccurate Y-Axis chart label for CBA. Note that files that OverviewMapStyled.js, Server.js, SampleCounties.js, AlgorithmDefaultParams.js, Helpers.js were renamed with capitalization to improve consistency with frontend naming conventions between files.

example of CBA chart label (previously YAxis label said kW which is incorrect)
Screen Shot 2020-11-05 at 7 49 59 PM
demonstrates the moved x-axis/yaxis labels so it's not overlapped by the ticks.
Screen Shot 2020-11-05 at 7 50 05 PM

### CODE REVIEW

File: frontend/src/Components/AlgorithmPage/AlgorithmPage.js [hunk 1]
```diff
@@ -51,6 +58,7 @@ class AlgorithmPage extends Component {
                                         title={ this.props.title }
                                         visualizeResults={ this.visualizeResults.bind(this) }
                                         setChartTitles={ this.setChartTitles.bind(this) }
+                                        checkCBA={ this.checkCBA.bind(this) }
```
Comment (jongoncalves) [inline frontend/src/Components/AlgorithmPage/AlgorithmPage.js]: Quick tip, if you write the `checkCBA` function in ES6 arrow syntax, you don't have to `bind`, it implicitly does it for you. No need to change anything though.

File: frontend/src/Components/Api/AlgorithmDefaultParams.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+export const loadForecastDefaultParams = {
+
```
Comment (jongoncalves) [inline frontend/src/Components/Api/AlgorithmDefaultParams.js]: 🔪 empty line

File: frontend/src/Components/Api/SampleCounties.js [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+// function (async/await)
+// call the function and return the result
+
+export const counties = {
+    "del norte": {
+        total_energy: Math.random(),
+        total_session: Math.random(),
+        peak_energy: Math.random(),
+    },
```
Comment (jongoncalves) [inline frontend/src/Components/Api/SampleCounties.js]: DRY up the object creation process here. No need to repeat all of this.

File: frontend/src/Components/Helpers/Helpers.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
+// Helper functions
+import axios from "axios";
+
+export function processResults(resultArr, isCBA) {
+    const data_to_visualize_all = [];
```
Comment (jongoncalves) [inline frontend/src/Components/Helpers/Helpers.js]: camelCase JS var

File: frontend/src/Components/Helpers/Helpers.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    if (!resultArr) {
+        resultArr = [];
+        isTimeSeries = false;
+    } else if (resultArr[0][0] !== undefined && resultArr[0][0].length === 0) {
+        isTimeSeries = false;
+    } else if (resultArr[0][Object.keys(resultArr[0])[0]][0].time) {
+        isTimeSeries = true;
+    } else {
+        isTimeSeries = false;
+    }
```
Comment (jongoncalves) [inline frontend/src/Components/Helpers/Helpers.js]: Extraneous conditionals. `isTimeSeries` is already set to `false`. No need to re-set it

File: frontend/src/Components/Helpers/Helpers.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    for (const field of fields) {
+        resultFlattened.push({
+            [field]: result[field],
+        });
+    }
+    return resultFlattened;
+}
+
+export async function checkFlowerTaskStatus (task_id) {
+    const task_res = await axios({
```
Comment (jongoncalves) [inline frontend/src/Components/Helpers/Helpers.js]: camelCase

File: frontend/src/Components/AlgorithmPage/AlgorithmPage.js (hunk 1 above)
Comment (avpeery) [reply frontend/src/Components/AlgorithmPage/AlgorithmPage.js]: Thanks for the tip! Is the arrow function the preferred method/better practice?

Comment (jongoncalves) [reply frontend/src/Components/AlgorithmPage/AlgorithmPage.js]: The arrow function is my preference and it seems that it's the direction the community is going.

File: frontend/src/Components/Api/SampleCounties.js (hunk 3 above)
Comment (avpeery) [reply frontend/src/Components/Api/SampleCounties.js]: This file isn't used anywhere anymore, so I will delete it.

**Comment by avpeery** [comment]
@jongoncalves @derins fixed initial commits. Applied Prettier to files, so some added changes, deleted some unused imports and unused files (SampleCounties). Ready for review

**Review by jongoncalves — APPROVED** [review]

**Review by derins — APPROVED** [review]

**Pull request merged**